go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/google/go-cmp v0.5.9 // indirect
 	github.com/inconshreveable/mousetrap v1.0.1 // indirect
+	github.com/jdx/go-netrc v1.0.0 // indirect
```

Comment:
Mind to rebase the branch and add some documentation so we can merge?

Could the path be configurable through some env variable? And then some test that does some url rewriting or something that requires no human interaction be created?